File: Neon Number/NEON.py
import math
num=int(input())
s=0;
square=int(math.pow(num,2));
while(square):
    s=s+square/10
    square=square/10
if(s==num):
    print("it is a neon number")
else:
    print("it is not a neon number")

File: Neon Number/neon.rs
use std::io;
fn main(){
    let mut num = String::new();
    io::stdin().read_line(&mut num).expect("Failed to read line");
    let num: u32 = num.trim().parse().expect("Please type a number!");
    
    let mut sum: u32 = 0;
    let mut squared: u32 = num.pow(2);
    while squared > 0 {
        let sum = sum + squared % 10;
        let squared = squared / 10;
    }
    if sum == num {
        println!("{} is a neon number", num);
    }
    else {
        println!("{} is not a neon number", num);
    }
}